### ISSUE TITLE
Genome size estimated by flowcytometer is smaller then canu assembly

### ISSUE THREAD

**Issue by amit4mchiba** [marbl/canu#1027]
Hi,

I am writing here to seek your views on my genome assembly project. I am working on a dipoloid plant genome, and trying to assemble its genome using 120x pacbio sequencing coverage. Before running Canu, i obtained flowcytomter based genome estimation, which suggested my genome as 3.01x bigger then Arabidopsis, which mean around 406Mb. When I performed Canu, my assembly ended up in 440Mb. I further checked my assembly by first performing arrow based polishing, then bionano based scaffolding, followed by HiC scaffolding, then correction through bionano again, then filling gap using Pacbio raw reads, polishing again through arrow, and finally through pilon using 90x genome coverage of illumina reads.  My assembly looks amazing, yet, final assembly is 440Mb, which is higher then flowcytometer estimation. I am not sure if some of repeats or wrong scaffolding has caused this increase or i should redo genome estimation. When I used kmer based genome estimation, suggested genome size is 448Mb. I am now tempted to believe this result but I do not know why i should not believe flowcytometer results. Please help me.

**Comment by skoren** [comment]
We've certainly seen genome size estimates be incorrect. It is also possible you have duplication due to haplotype variation in your genome. There is some information on the FAQ on how to check for this duplication using purge_haplotigs. You can also use core genes to check duplication in the assembly. If you don't find duplication and the purge_haplotigs coverage plot indicates a single peak, then I'd trust the assembly size.

**Comment by amit4mchiba** [comment]
Thank you so much for your reply. I did checked my assemblies using BUSCO, and here are the results-
# BUSCO version is: 3.0.1 
# The lineage dataset is: embryophyta_odb9 (Creation date: 2016-02-13, number of species: 30, number of BUSCOs: 1440)
# BUSCO was run in mode: genome
	
C:93.8%[S:91.7%,D:2.1%],F:1.5%,M:4.7%,n:1440
1351 Complete BUSCOs (C)
1321 Complete and single-copy BUSCOs (S)
30   Complete and duplicated BUSCOs (D)
22   Fragmented BUSCOs (F)
67   Missing BUSCOs (M)
1440 Total BUSCO groups searched

Only 2.1% duplicated genes compared to 91.7% of complete BUSCO matches. This suggests not major duplication for this assembly. I will check purge_haplotigs and then as your suggested, will be able to see if we can trust this assembly. I did run nucmer to see my assembly feature and plots looked nice to me. I will check again.

Thank you so much

**Comment by andy-j-king** [comment]
Hi Amit

It's worth bearing in mind genome estimates by flow cytometry are just that.  If you check all the flow cytometry details for a particular species on Kew's C-value database, you'll see there's usually a large standard deviation.  There a a number of reasons for this, including not having precise genome size values for the standards themselves, and differences in staining between nuclei preparations from different organisms.

I won't worry about a 10% difference in the genome sizes estimated from flow cytometry and genome assembly.

**Comment by alslonik** [comment]
Hi Amit, 

May I ask a few questions about your assembly, just because I am doing something very similar - diploid plant genome ~350-400 Mb is currently under assembly on canu. I had the same coverage to start with - X120. 
-How long did it take and on what machine? (just so that i will estimate when my run is going to finish)
-Is your genome has high percentage of repetative sequences?

Thanks, 
Alex

**Comment by amit4mchiba** [comment]
Hi Andy,

Thanks a lot. I agree with you. Actually k-mer analysis get me close to the assembly size that i finally obtained but flow-cytometer results were quite different. I performed purge_haplotigs from the draft assembly, but no change. So, I am now assuming that all is good and flow-cytometer reading were quite off for some reason. Thank you for your reply.

Hi Alex,

I have access to my own server, which has 500GB ram, 50TB hard-disk, 18 cores nad lattest Intel processor. Its good to have a nice server, as it does things in short time. For my case, when I run Canu for my 120x genome coverage, it resulted an amazing assembly, with contig-N50 as 9.3Mb, and I got my result within 2-days.
My genome does not have high percentage of repetative sequences, its more homozygous and nice to assemble. I followed mostly as described on Goat-genome. I tried Pacbio, followed by Bionano, then HiC, then gap filling using Pbjelly and finally Arrow and pilon based polishing, Like goat genome, I found advantage in using pacbio+bionano for HiC analysis.

Hope this helps.
